src/views/Workflow.vue

```diff
@@ -97,16 +97,23 @@ export default {
       TreeView,
       TableView,
       GraphView
-    ]
+    ],
+    // environment e.g. 'PRODUCTION'
+    environment: process.env.VUE_APP_SERVICES === 'offline' ? 'OFFLINE' : process.env.NODE_ENV.toUpperCase()
   }),
   created () {
     // We need to load each view used by this view/component.
     // See "local-registration" in Vue.js documentation.
     this.views.forEach(view => {
       this.$options.components[view.name] = view
     })
-  },
-  computed: {
+
+    if (this.environment !== 'PRODUCTION') {
+      // dynamically load development views that we don't want in production
+      import('@/views/SimpleTree').then((SimpleTreeView) => {
+        this.views.push(SimpleTreeView.default)
+      })
+    }
```

Comment:
Please check that the SimpleTree is present when the UI is built in dev mode (i.e. `build:watch`) but not when run in production mode (i.e. `build`).

Reply:
Checked, all good.

Reply:
me too